app/src/main/java/org/fossasia/openevent/general/event/EventService.kt

```diff
@@ -21,6 +21,20 @@ class EventService(private val eventApi: EventApi, private val eventDao: EventDa
         }
     }
 
+    fun getSearchEvents(eventName: String): Flowable<List<Event>> {
+        val eventsFlowable = eventDao.getSearchEvents()
+        return eventsFlowable.switchMap {
```

Comment:
Doesn't make sense to switchmap when you're not using the original flowable anywhere